### PR TITLE
[v1.0beta1] devel -> master

### PR DESCRIPTION
**Pull request by diegoferigo**
Preparation for a new release

### CODE REVIEW

**Comment by diegoferigo** [comment]
@traversaro Given the last PRs that fixed and simplified the installation, despite there are not many new features, I think we should make a new release. Let me know if you spot last-minute typos.

File: README.md [hunk 1]
```diff
@@ -114,36 +142,74 @@ At the time of writing, Gym-Ignition offers the following features:
...
-   module_path=$(python -c "import gym_ignition ; print(gym_ignition.__path__[0])" | grep gym_ignition)
-   export IGN_GAZEBO_SYSTEM_PLUGIN_PATH=${module_path}/plugins
+   sudo sh -c 'echo "deb http://packages.osrfoundation.org/gazebo/ubuntu-stable `lsb_release -cs` main" > /etc/apt/sources.list.d/gazebo-stable.list'
+   wget http://packages.osrfoundation.org/gazebo.key -O - | sudo apt-key add -
+   sudo apt update
+   sudo apt install ignition-gazebo2 libignition-gazebo2-dev
    ```
 
-After these steps, you should be able to execute the example [`launch_cartpole.py`](examples/python/launch_cartpole.py).
+   We currently require the development version, that will be ABI-compatible with the upcoming **Citadel** release.
```
Comment (traversaro) [inline README.md]: I am a bit confused, I thought that Citadel was ignition-gazebo3 .

Comment (diegoferigo) [reply README.md]: Yes, Citadel is going to be ignition-gazebo3. I checked my local branches and, despite Blueprint has been released before the summer, we still compile against branches ABI-compatible with it.

This check made me realize that we can still use the metapackage `ignition-blueprint` to install everything, and the current instructions actually do not install some (maybe not important) dependency such as `libignition-launch` and libwebsockets.

Comment (diegoferigo) [reply README.md]: Corrected in 6e0702aea5070dde551be7d78cd2374ab3c863b8

**Pull request merged**